iOS/Flows/Room/RoomHeaderView.swift

```diff
@@ -31,16 +39,26 @@ class RoomHeaderView: BaseView {
         }
         
         self.addSubview(self.focusCircle)
-        if let status = User.current()?.focusStatus {
-            self.focusCircle.set(backgroundColor: status.color)
-        } else {
-            self.focusCircle.set(backgroundColor: FocusStatus.focused.color)
-        }
+        self.update(status: User.current()?.focusStatus)
         
         self.addSubview(self.button)
         
         self.addSubview(self.roomNavButton)
         self.roomNavButton.configure(for: .outer)
+        
+        PeopleStore.shared.$personUpdated.mainSink { person in
```

Comment:
Update the circle color when focus status changes.

Reply:
We'll need an unowned reference or else this is a memory leak